docs/gallery/interactive/pyfar_plotting.ipynb

```diff
@@ -0,0 +1,423 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pf.plot.time_2d(impulses, unit='ms')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Styling plots\n",
+    "\n",
+    "From the plots above, you might have noticed that the plots look different from the default matplotlib style.\n",
+    "This is because pyfar changes the default style of matplotlib towards a more convenient style for acoustic research.\n",
+    "Some of the changes include:\n",
+    "\n",
+    "- Grid enabled by default\n",
+    "- Frequency axis with adapted tick locations\n",
+    "- Custom color cycle, wich also overwrites the default matplotlib colors (e.g. `'r'` results in the red defined by [`pyfar.plot.color`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.plot.html#pyfar.plot.color))\n",
+    "\n",
+    "Furthermore, pyfar includes two pyfar-specific plot styles: _light_ and _dark_.\n",
+    "The default style is _light_.\n",
+    "\n",
+    "The pyfar plot functions accept a `style` argument to change the style."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal = pf.Signal([0, 1, 0, -1], 44100)\n",
+    "\n",
+    "pf.plot.time(signal, style=\"dark\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The overall style can be temporarily changed using `pyfar.plot.context` which is a wrapper around `matplotlib.context`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with pf.plot.context(style=\"dark\"):\n",
+    "    # this plot uses the dark style\n",
+    "    fig, ax = plt.subplots(1, 1)\n",
+    "    pf.plot.time(signal, ax=ax)\n",
+    "\n",
+    "# plots outside the `with` context would use the standard style again"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This overall change in style can also be made permanent with the `pf.plot.use()` function like this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pf.plot.use(style=\"dark\")\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "pf.plot.time(signal, ax=ax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interactive plots\n",
+    "\n",
+    "pyfar provides keyboard shortcuts for switching plots, zooming in and out, moving along the x and y-axis, and for zooming and moving the range of color maps.\n",
+    "To do this, you need to use an interactive matplotlib backend.\n",
+    "This can for example be done by including\n",
+    "\n",
+    "```python\n",
+    "%matplotlib qt\n",
+    "```\n",
+    "\n",
+    "or\n",
+    "\n",
+    "```python\n",
+    "matplotlib.use('Qt5Agg')\n",
+    "```\n",
+    "\n",
+    "in your code.\n",
+    "\n",
```

Comment:
I think having `%matplotlib qt` and `%matplotlib ipympl` would be sufficient here :)
```suggestion
```